scripts/configure-multi-node-aio.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
The only difference is it calls the configure-multi-node-aio.yml playbook and sets the TARGET variable to mnaio if it hasn't been already. We'll set TARGET in the job, but I thought it would be easier for people to use outside of the gate if the script also set it.